jsoniter-scala/src/main/scala/com/free2move/geoscala/jsoniter_scala.scala

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2019 GHM Mobile Development GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.free2move.geoscala
+
+import com.github.plokhotnyuk.jsoniter_scala.core._
+import com.github.plokhotnyuk.jsoniter_scala.macros._
+import scala.annotation.nowarn
+
+object jsoniter_scala {
```

Comment:
do we need the `_scala` in the package name?

Reply:
Yes, [jsoniter](https://jsoniter.com/) is quite a different beast of software, and jsoniter-scala doesn't depend on it for ages